kubernetes/samples/scripts/create-weblogic-domain/model-in-image/utils/wl-pod-wait.sh

```diff
@@ -5,8 +5,8 @@
 #
 # This is a utility script that waits until a domain's pods have all exited,
```

Comment:
I wonder if the description here should match the description in `common-mounts.md`. Both descriptions look good to me but it might be good to have the same wordings in both places.

Reply:
done